Procfile

```diff
@@ -1,4 +1,4 @@
 web: WEBPACK_PORT=${WEBPACK_PORT:-3035} bundle exec rackup config.ru --port ${PORT:-3000} --host ${FOREMAN_HOST:-${HOST:-localhost}}
 worker: bundle exec good_job start
-js: WEBPACK_PORT=${WEBPACK_PORT:-3035} yarn webpack --watch
+js: WEBPACK_PORT=${WEBPACK_PORT:-3035} ORIGIN_PORT=${ORIGIN_PORT:-3000} yarn webpack --watch
```

Comment:
The reason for creating a separate environment variable copying `PORT` is that the behavior of Foreman is to assign a different port value for each process (see https://github.com/ddollar/foreman/pull/110, https://github.com/ddollar/foreman/issues/714). If we just passed `PORT`, it would actually assign `3200`, since this is the third process in the Procfile.